### DISCUSSION TITLE
implement currentFPS for SRT

### DISCUSSION THREAD

**Discussion by ruirodrigues-eyenov** [HaishinKit/HaishinKit.swift#1324] [Idea]
Hello
I was looking in how to have [currentFPS](https://github.com/shogo4405/HaishinKit.swift/blob/main/Sources/Net/NetStream.swift#L241) for updated SRTStream similar to how it is implemented in RTMPStream/RTMPConnection.
However the `currentFPS` in NetStream setter is internal so SRTStream cannot write into this property.

One option is to make it public, similar to `readyStateWillChange(to:)` , or use the new [package access modifier](https://github.com/apple/swift-evolution/blob/main/proposals/0386-package-access-modifier.md) available in swift 5.9.

**Comment by shogo4405** [comment]
Creating a method for currentFPS seems excessive. It might be a good idea to remove the definition of NetStream.currentFPS and define it in SRTStream and RTMPStream instead.
